microshift_getting_started/microshift-architecture.adoc

```diff
@@ -10,7 +10,24 @@ Learn the specifics of {product-title} architecture including design intent, how
 
 [id="microshift-architectural-design_{context}"]
 == Architectural design
-{product-title} is a single-node container orchestration runtime designed to extend the benefits of using containers for running applications to low-resource edge environments. {product-title} contains only the APIs and features essential to operating in edge and small form factor computing environments. The following operational differences from {OCP} can help you understand where {product-title} can be deployed:
+{product-title} is a single-node container orchestration runtime designed to extend the benefits of using containers for running applications to low-resource edge environments. {product-title} contains only the APIs and features essential to operating in edge and small form factor computing environments. {product-title} is tasked with only the following Kubernetes cluster services:
+
+* Networking
+* Ingress
+* Storage
+* Helm
+
+{product-title} also provides the following Kubernetes functions:
+
+* Orchestration
+* Security
+
+{product-title} is optimized when used with a compatible operating system, such as {op-system-ostree-first}. When used together, {product-title} and {op-system-ostree-first} form {op-system-bundle}. {product-title} can also be deployed as a component by users of {OCP}.
+
+.{product-title} as part of {op-system-bundle}.
+image::microshift-RHDE-Tech-Overview.png[<{product-title} is tasked with only the Kubernetes cluster services networking, ingress, storage, helm, with additional Kubernetes functions of orchestration and security.>]
```

Comment:
"services" again, see earlier comment

Reply:
I realized the graphic actually says "Kubernetes services," which is why I used the term. Is it not accurate on the slide? I can edit that as well. I was thinking "capabilities."

Reply:
We'll leave the "services" on the graphic, but we can tweak the technical language in the docs to "capabilities."